src/include/distributed/metadata_utility.h

```diff
@@ -34,6 +34,12 @@
 #define PG_RELATION_SIZE_FUNCTION "pg_relation_size(%s)"
 #define PG_TOTAL_RELATION_SIZE_FUNCTION "pg_total_relation_size(%s)"
 #define CSTORE_TABLE_SIZE_FUNCTION "cstore_table_size(%s)"
+#define SHARD_SIZES_QUERY \
+	"select table_schema || '.' || table_name, pg_relation_size" \
+	"('\"'||table_schema||'\".\"'||table_name||'\"') from " \
```

Comment:
this seems quite sketchy, e.g. it adds quotes without actually doing any escaping.

this should probably use pg_class